Providers/RustSyncManager.swift

```diff
@@ -189,8 +189,7 @@ public class RustSyncManager: NSObject, SyncManager {
         }
 
         if canSendUsageData() {
-            let gleanHelper = GleanSyncOperationHelper()
```

Comment:
It looks like `GleanSyncOperationHelper` class could be removed as well?

Reply:
@lmarceau That class is defined in the [SyncTelemetryUtils.swift](https://github.com/mozilla-mobile/firefox-ios/blob/main/Sync/SyncTelemetryUtils.swift#L12) file, which I'm deleting.

Reply:
Oh sorry, I missed that! Thank you!

Reply:
No problem! 😊